### ISSUE TITLE
Python 3.10 build

### ISSUE THREAD

**Issue by d-cogswell** [v1kko/daetools#3]
I saw a new version (1.9.2) on Conda Forge that looks like it has a Python 3.10 build. Is that correct?

**Comment by v1kko** [comment]
yes thats correct, conda forge seemed to be the best place to put daetools, see the recipe here: https://github.com/conda-forge/daetools-feedstock . If daetools compile chain does not break this means it will be available automatically when conda-forge decides to recompile for newer python versions👍

**Comment by d-cogswell** [comment]
Great! Thanks for configuring this.
